.github/workflows/release.yml

```diff
@@ -56,7 +54,7 @@ jobs:
     - name: Set up Go 1.x
       uses: actions/setup-go@v5
       with:
-        go-version: 1.21
+        go-version: 1.23
```

Comment:
Looks like this was missed in the Go version change by accident.

Reply:
ah good catch, thanks for fixing this!